src/theme.ts

```diff
@@ -12,7 +12,7 @@ interface PaletteIntensityOptions {
 
 declare module '@mui/material/styles/createPalette' {
   interface Palette {
-    onSurface: PaletteIntensityOptions;
+    onSurface: Required<PaletteIntensityOptions>;
```

Comment:
What is the purpose of this? Asking because I've been wrestling the palette every now and then and have realized that I don't really understand how MUI works in that regard.

Reply:
The type PaletteIntensityOptions is defined higher up in this file and has all its properties as optional, which made typescript complain when I wanted to access one of the properties in my component. So I fixed by making them required on this object, so TS would know there was always a "disabled"-property.